common/config/beachball/changelog-config.ts

```diff
@@ -51,8 +52,7 @@ export const config: BeachballConfig = {
       name: "@azure/communication-react and its packlets",
       include: "packages/*",
       disallowedChangeTypes: [
-        "major",
-        "none"
+        "major"
```

Comment:
Should we continue using patch/minor for this instead of adding none back (as the changes will be added to the changelog file)?

Reply:
Updated